dafni_cli/commands/helpers.py

```diff
@@ -43,6 +46,33 @@ def cli_get_latest_dataset_metadata(session: DAFNISession, version_id: str) -> d
         raise SystemExit(1) from err
 
 
+def cli_select_dataset_files(
+    dataset_metadata: DatasetMetadata,
+    files: Optional[List[str]],
```

Comment:
Is this actually optional, or do we just mean it can be set to None (but the argument must be provided in the call)?

Reply:
Yes exactly, it uses None to indicate no files are specified, but we always require the parameter to be given.